.prettierignore

```diff
@@ -1,11 +1,10 @@
-.eslintrc.js
```

Comment:
This was done due to the `keep-sorted` rule not being set-up properly back in the day. I fixed that a while ago.